astropy/cosmology/scalar_inv_efuncs.pyx

```diff
@@ -1,5 +1,5 @@
+#cython: language_level=3, boundscheck=False
```

Comment:
The old stuff was `#cython boundcheck=False` but according to the Cython doc, `boundcheck` might be a typo. Not sure if the colon is necessary, but no harm in following official examples? @aconley , what do you think?

Reply:
Almost certainly a mistake, so I say go for it.  That's what the tests are for, right?